impl-base/src/main/java/org/jboss/shrinkwrap/impl/base/IdentifiableArchiveImpl.java

```diff
@@ -47,7 +39,5 @@ public String getId() {
     @Override
     public void setId(final String id) throws IllegalArgumentException {
         this.getArchive().setId(id);
-
```

Comment:
Probably just syntax fix from the IDE itself, I think we can keep these lines just to keep the diff clean as removing them doesn't really correspond with removing loggers